src/com/palmergames/bukkit/towny/hooks/LuckPermsContexts.java

```diff
@@ -28,10 +31,14 @@ public class LuckPermsContexts implements ContextCalculator<Player> {
 	private static final String INSIDEOWNPLOT_CONTEXT = "towny:insideownplot";
 	private static final String TOWN_RANK_CONTEXT = "towny:townrank";
 	private static final String NATION_RANK_CONTEXT = "towny:nationrank";
+	private static final String TOWN_CONTEXT = "towny:town";
+	private static final String NATION_CONTEXT = "towny:nation";
 	
 	private static final List<String> booleanContexts = Arrays.asList(RESIDENT_CONTEXT, MAYOR_CONTEXT, KING_CONTEXT, INSIDETOWN_CONTEXT, INSIDEOWNTOWN_CONTEXT, INSIDEOWNPLOT_CONTEXT);
 	private static final String nationRankContext = NATION_RANK_CONTEXT;
 	private static final String TownRankContext = TOWN_RANK_CONTEXT;
```

Comment:
I also noticed incorrect naming convention here. Should I fix this in same PR (anyway doing contexts)

Reply:
Sure

Reply:
These fields have no use, feel free to remove them

Reply:
Forgot they are private, thought of compatibility issues lol